libafl_frida/Cargo.toml

```diff
@@ -28,10 +28,10 @@ hashbrown = "0.12"
 libloading = "0.7"
 rangemap = "1.0"
 frida-gum-sys = { version = "0.3", features = [ "auto-download", "event-sink", "invocation-listener"] }
-frida-gum = { version = "0.6.5", features = [ "auto-download", "event-sink", "invocation-listener"] }
+frida-gum = { version = "0.7.1", features = [ "auto-download", "event-sink", "invocation-listener"] }
 regex = "1"
 dynasmrt = "1.2"
-capstone = "0.10"
+capstone = "0.11.0"
```

Comment:
Why 0.11.0 and not 0.11?

Reply:
https://crates.io/crates/capstone says 0.11.0, so I thought it is better

Reply:
But usually the last digit shouldn't change the api, just patch bugs

Reply:
> Then do it at both places

aah, sorry my mistake